app/src/renderer/vuex/modules/wallet.js

```diff
@@ -64,10 +64,16 @@ export default ({ commit, node }) => {
       dispatch('queryWalletSequence')
       dispatch('queryWalletHistory')
     },
-    async queryWalletBalances ({ state, commit }) {
+    async queryWalletBalances ({ state, rootState, commit }) {
       let res = await node.queryAccount(state.key.address)
       if (!res) return
       commit('setWalletBalances', res.data.coins)
+      for (let coin of res.data.coins) {
+        if (coin.denom === rootState.config.bondingDenom) {
+          commit('setAtoms', coin.amount)
```

Comment:
why are atoms treated differently than other denoms here?

Reply:
Could be cleaned up later, but we're using the 'atoms' value elsewhere in the app for bonding code. This is just a product of the original bonding code using a an atom balance separate from the wallet.